astropy/modeling/tests/test_bounding_box.py

```diff
@@ -686,7 +686,7 @@ def test__set_outputs_unit(self):
         bounding_box = self.BoundingDomain(mk.MagicMock())
 
         # set no unit
-        assert 27 == bounding_box._set_outputs_unit(27, None)
+        assert bounding_box._set_outputs_unit(27, None) == 27
```

Comment:
Why the next test on line 692 was not swapped?
```python
assert bounding_box._set_outputs_unit(27, u.m) == 27 * u.m
```

Reply:
Yes, it should be. However, ``27* u.m`` is the class instantiation ``Quantity(27, u.m)`` so the linter has no way to determine which is the test and which the expected result.

Reply:
There are two options: 1) manual correction, 2) assigning expected results to variables. The latter is often how ``pytest.mark.parametrize`` is used and the linter understands this code pattern.

Reply:
✅ Made the change